### DISCUSSION TITLE
Suggestion of notification using voice call

### DISCUSSION THREAD

**Discussion by jihyungSong** [cloudforet-io/community#3] [Ideas]
**Propose Date** : 2022-11-10
**Proposer** : @jihyungSong  @mintloveschoco

---

Currently, Cloudforet can deliver event information to users through various channels to the Notification service. (https://github.com/cloudforet-io/notification) 

The currently available notification protocol channels are as follows.

* [Email](https://github.com/cloudforet-io/plugin-email-noti-protocol)
* [Slack](https://github.com/cloudforet-io/plugin-slack-noti-protocol)
* [Telegram](https://github.com/cloudforet-io/plugin-telegram-noti-protocol)
* [SMS (Short Message) from MZC](https://github.com/cloudforet-io/plugin-mzc-sms-noti-protocol)

In addition to this, I propose **voice call** as a new notification channel.

In case of an urgent alarm event, it is a way to make sure the user is aware of an emergency through a direct phone call rather than a text-based delivery method.

In addition, this will be implemented based on AWS so that any AWS user can build and use itself (such as `cloudformation`), and will be implemented it the new Plugin. 

Based on `Amazon Connect`, a cloud contact center service, We plan to implement it using the `Amazon Polly` service that can easily text to speech.

The new repository name will be named `plugin-voicecall-noti-protocol` according to the repository naming rules.

**Comment by ImMin5** [comment]
I'm looking forward to new feature 👍

**Comment by jihyungSong** [comment]
New Repository is created [plugin-voicecall-noti-protocol](https://github.com/cloudforet-io/plugin-voicecall-noti-protocol)
